web/src/search/input/SearchPage.tsx

```diff
@@ -236,7 +253,7 @@ export const SearchPage: React.FunctionComponent<Props> = props => {
                                         <p>repo:r@*refs/heads/ (all branches)</p>
                                     </dd>
                                 </dl>
-                                <dl>
+                                <dl className="col-sm-5 col-md-12 col-lg-12 col-xl-6">
```

Comment:
Same question here - `md` is wider than `sm`?
```suggestion
                                <dl className="col-sm-5 col-md-12 col-xl-6">
```

Reply:
fixed